content/run.md

```diff
@@ -149,13 +149,13 @@ For dedicated single-cell simulation in order to compare with site-level observa
 There are dedicated tools and workflow for running single-cell simulations over Norwegian ecological observation sites using CLM and CLM-FATES, please check [NorESM_LandSites_Platform](https://github.com/NorESMhub/NorESM_LandSites_Platform) and follow the instructions there.
 
 ## Run a single cell case by creating surface data
-CLM also supports running single-point simulations by creating single point surface data using python [script](https://github.com/ESCOMP/CTSM/blob/master/python/ctsm/subset_data.py) from NCAR. Note that you need to change the global input data directory variable inside the script (standard global CTSM dataset) as below:
-`dir_inputdata='/cluster/shared/noresm/inputdata/'` (few lines after region.create-tag()).
+CLM also supports running single-point simulations (referring to version ctsm5.1.dev061-3-g7a85024) by creating single point surface data using python [script](https://github.com/devarajun/python-scripts/blob/main/subset_data.py). Make sure the global inputdata directory path is correct:
```

Comment:
We shouldn't use personal forks here. Let's either refer to the ESCOMP repo or our own fork under MetOs account.